src/Parser.php

```diff
@@ -60,7 +60,9 @@ public function getEnvironment(): Environment
 
     public function getVarName(): string
     {
-        return \sprintf('__internal_parse_%d', $this->varNameSalt++);
+        trigger_deprecation('twig/twig', '3.15', 'The "%s()" method is deprecated.', __METHOD__);
+
+        return \sprintf('_n%d', $this->varNameSalt++);
```

Comment:
why changing the name of the variable being generated in this deprecated method ?

Reply:
It makes reading the output code easier to read. But you're right, it's not needed.